packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3371,6 +3372,7 @@ const ruleset = {
             description: "Extension resources are always considered to be proxy and must not be of the type tracked.",
             message: "{{error}}",
             severity: "error",
+            stagingOnly: true,
```

Comment:
I am curious, is this some leftover from previous change that was not generated into the `.js` file from `.ts` file? Also, btw. rule name is lower-cased: `trackedExtensionResourcesAreNotAllowed`. Should be upper-cased. But we can fix it in another PR.